### PR TITLE
Personal blog using Hugo and Webiny

### PR DESCRIPTION
**Pull request by AmenyaZ**

### CODE REVIEW

**Review by endymion1818 — CHANGES_REQUESTED** [review]
Thanks @AmenyaZ for this code, please could you remove the node_modules you've checked in? You will need to add a `.gitignore` to the project root, add the **node_modules** folder and then delete it from the git history and push again.

**Comment by AmenyaZ** [comment]
I made a second push. Please check the commit

File: tutorials/hugo-blog/config.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+baseURL = 'https://dgpcvq8nrbk6c.cloudfront.net/cms/read/en-US'
```
Comment (endymion1818) [inline tutorials/hugo-blog/config.toml:1]: Please remove this too

**Review by endymion1818 — CHANGES_REQUESTED** [review]
Nearly there!

**Pull request closed without merging**